### PR TITLE
Update GridView.xml

### PR DESCRIPTION
**Pull request by SimpleSamples**
## Summary

I just added that null is returned if nothing is selected.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: xml/System.Web.UI.WebControls/GridView.xml [hunk 1]
```diff
@@ -5872,7 +5872,7 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a reference to a <see cref="T:System.Web.UI.WebControls.GridViewRow" /> object that represents the selected row in the control.</summary>
-        <value>A <see cref="T:System.Web.UI.WebControls.GridViewRow" /> that represents the selected row in the control.</value>
+        <value>A <see cref="T:System.Web.UI.WebControls.GridViewRow" /> that represents the selected row in the control or null if nothing is selected.</value>
```
Comment (rpetrusha) [inline xml/System.Web.UI.WebControls/GridView.xml]:
```suggestion
        <value>A <see cref="T:System.Web.UI.WebControls.GridViewRow" /> that represents the selected row in the control, or `null` if nothing is selected.</value>
```

**Comment by SimpleSamples** [comment]
I assume I have done what I need to do.

**Comment by rpetrusha** [comment]
Yes, you have, @SimpleSamples. I'll merge your PR now. Your changes should be live on docs.microsoft.com in the next day or two.

**Comment by rpetrusha** [comment]
Sorry, @SimpleSamples. I accidentally closed your PR in commenting. I've reopened it, but merging has to wait until an internal build completes.

**Pull request merged**